cdap-ui/app/cdap/components/RulesEngineHome/RuleBookDetails/RuleBookDetails.scss

```diff
@@ -99,7 +100,7 @@ $rule-book-create-fields-required-color: $cdap-red-text;
   }
 
   .rule-book-content {
-    height: calc(100% - 100px);
+    height: calc(100% - #{$rule-book-name-header-height});
```

Comment:
I think you meant `$` instead of `#`

Reply:
no, you can't just put a variable in a `calc` function